ckan/logic/validators.py

```diff
@@ -888,3 +889,16 @@ def callable(value):
             raise Invalid(_('Value must be one of {}'.format(list_of_value)))
         return value
     return callable
+
+
+def json_object(value):
+    ''' Make sure value can be serialized as a JSON object'''
+    if value is None or value == '':
+        return
+    try:
+        if not json.dumps(value).startswith('{'):
+            raise Invalid(_('plugin_extras should be valid a JSON object'))
+    except ValueError as e:
+        raise Invalid(_('Could not parse plugin_extras as a valid JSON object'))
```

Comment:
the field name is already included as the key of the errors dict, no need to repeat it here (and removing plugin_extras from the error lets us use this validator elsewhere)